gems/smithy/spec/support/client_helper.rb

```diff
@@ -4,95 +4,18 @@
 
 module ClientHelper
```

Comment:
I went back and forth on this but I'm happy with the result - I went too far with the refactor and dialed it back some. I think that the smithy spec_helper/client_helper should work only on fixtures. The purpose of those is to verify that the generated code/files are correct. I prefer the explicit module name declaration in the tests for smithy's spec_helper. I think having smithy-client rely on another gem's spec helper was funky, so I pulled out sample_shapes into smithy-clients client_helper, and I think it's ok to have two. At this point, when using this helper, we can assume the generated code is correct. The quick runtime client generation is great for smithy-client tests and does not need fixtures in my opinion, as we probably want to change model.